backend/app/api/endpoints/chapters.py

```
@@ -0,0 +1,592 @@
+        )
+
+        return {
+            "book_id": book_id,
+            "chapter": new_chapter,
+            "chapter_id": new_chapter["id"],
+            "success": True,
+            "message": "Chapter created successfully",
+        }
+    except ValueError as e:
+        if "not found" in str(e).lower():
+            raise HTTPException(status_code=404, detail="Book not found")
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)